csireverseproxy/pkg/config/config.go

```diff
@@ -566,6 +638,122 @@ func (pc *ProxyConfig) ParseConfig(proxyConfigMap ProxyConfigMap, k8sUtils k8sut
 	return nil
 }
 
+// ParseConfigFromSecret - Parses a given proxy secret
+func (pc *ProxyConfig) ParseConfigFromSecret(proxySecret ProxySecret, k8sUtils k8sutils.UtilsInterface) error {
+	// TODO: Need to get port information dynamically, Hardcoded for now
```

Comment:
Address this TODO in this PR or the PR going into main?

Reply:
Code changes already made, but I will merge this in the next iteration. I have code changes that are not yet pushed that contains changes for reading common config map for port and log information. It needs a bit more testing before i can commit those changes to the branch.